test/jdk/ProblemList.txt

```diff
@@ -475,7 +475,6 @@ java/awt/event/MouseEvent/ClickDuringKeypress/ClickDuringKeypress.java 8233568 m
 java/awt/event/KeyEvent/DeadKey/DeadKeyMacOSXInputText.java 8233568 macosx-all
 java/awt/event/KeyEvent/DeadKey/deadKeyMacOSX.java 8233568 macosx-all
 java/awt/TrayIcon/RightClickWhenBalloonDisplayed/RightClickWhenBalloonDisplayed.java 8238720 windows-all
-java/awt/PopupMenu/PopupMenuLocation.java 8259913,8315878 windows-all,macosx-aarch64
```

Comment:
[JDK-8259913](https://bugs.openjdk.org/browse/JDK-8259913) with `windows-all` has to stay in the problem list because the test fails on High DPI systems. I ran it locally on a screen with 200%, and the test failed.

The test may fail on mach5 hosts with High DPI displays too. I guess you were lucky to run the test on a regular display.